src/layers/core/grid-cell-layer/grid-cell-layer-vertex-64.glsl.js

```diff
@@ -58,14 +57,19 @@ float isPicked(vec3 pickingColors, vec3 selectedColor) {
 
 void main(void) {
   vec4 instancePositions64xy = vec4(
-    instancePositions.x + (positions.x + 1.0) * lonOffset / 2.0,
```

Comment:
This didn't get changed during our last minute API change to the grid layer 32-bit